[MTM-64106] CA feature release notes, information about model change (#3811)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-346-0-certificate-authority-feature-generally-available.md

- Perform legitimacy checks, as defined by each tenant
- Issue signed X.509 certificates trusted by the device tenant

For more details about this feature refer to [Certificate Authority (CA)](/device-certificate-authentication/certificate-authority).

{{< c8y-admon-info >}}

> For environments where the feature was used during the Public Preview, it should be noted that the certificate model was updated in version 2025.279.0.

Reviewer: ```suggestion
For environments that used this feature during Public Preview, note that the certificate model was updated in version 2025.279.0.
```

- Issue signed X.509 certificates trusted by the device tenant

For more details about this feature refer to [Certificate Authority (CA)](/device-certificate-authentication/certificate-authority).

{{< c8y-admon-info >}}

> For environments that used this feature during Public Preview, note that the certificate model was updated in version 2025.279.0.

Reviewer 2: ```suggestion
If you used this feature during the Public Preview, note that the certificate model was updated in version 2025.279.0.
```

> Provisioned certificates obtained for devices prior to this version may not be compatible with the current release.

Reviewer 3: is there any uncertaintly as to whether the provision certificates are compatible?  If not then this should be: will not be compatible with the current release.

Author: devices will be able to connect, but they will not be able to refresh the certificate, so it will only be valid for the time for which it was initially generated.

> It is required to remove such devices and provision new certificates to ensure compatibility and proper operation.

Reviewer 3: Do the devices need to be completely de-registered or just have new certificates issued?

Author: The whole device's representation in Cumulocity must be deleted.
Otherwise, during reenrolling new certificate exception will occur, and the device will not be able to generate a new certificate.

Reviewer 3: In that case I would proposed we make it much clearer with a banner or some such with the words:

Migration from Public Preview to General Availability - action required.  As part of the move to General Availability you need to remove all the devices you have registered under the public preview and re-register them.  All such devices will continue to be able to connect but none of the other capabilities of the certificate lifecycle management will be available until they are re-registered.  

If we do not put such a banner on the documentation we will potentially have issues being raised for the foreseeable future which is not a good customer experience.

Author: Created PR https://github.com/Cumulocity-IoT/c8y-docs/pull/3825

> It is recommended to remove such devices and provision new certificates to ensure compatibility and proper operation.

Reviewer 2: I think we must be more explicit here. It is not only "recommended" but "needed".
```suggestion
It is required to remove such devices and provision new certificates to ensure compatibility and proper operation.
```

Author: You are right, to avoid unclear situations and exceptions, they should remove such devices
